openassessment/xblock/ui_mixins/mfe/ora_config_serializer.py

```diff
@@ -217,9 +217,7 @@ class OraBlockInfoSerializer(Serializer):
 
     def get_baseAssetUrl(self, block):
         # pylint: disable=protected-access
-        return block.get_base_url_path_for_course_assets(
```

Comment:
Why the filename change for this file?

Reply:
I've been splitting the serializers by the response shape they are returning for. This is helping (me at least) keep things separated, since otherwise this would be a 1k+ line file by the time we're done with it which makes finding things harder.